### PR TITLE
process: update ci project used by kokoro

### PR DESCRIPTION
**Pull request by Shabirmean**
Import changes from https://github.com/googleapis/java-container/pull/628 into the LTS branch `2.2.x`

### CODE REVIEW

**Review by olivi-eh — APPROVED** [review]
LGTM

File: owlbot.py [hunk 1]
```diff
@@ -21,4 +21,8 @@
     s.move(library)
 
 s.remove_staging_dirs()
-java.common_templates()
+java.common_templates(excludes=[
+    '.kokoro/presubmit/graalvm-native.cfg',
```
Comment (Neenu1995) [inline owlbot.py:25]: You may want to exclude the graalvm-native cfg from the exclusion list. The Graalvm team is still working on it and may require changes to the file.

Comment (Shabirmean) [reply owlbot.py:25]: @Neenu1995 thank you for the comment. This is a cherrypick from what we have on `main`. Should that change go into `main` as well if we are making that change in the `2.2.x` branch?

**Review by Neenu1995 — APPROVED** [review]

File: owlbot.py (hunk 1 above)
Comment (bourgeoisor) [reply owlbot.py:25]: More context, it comes from this PR: https://github.com/googleapis/java-container/pull/628

Comment (bourgeoisor) [reply owlbot.py:25]: The way I understand it, if we don't exclude the graalvm-native config, then changes to it are going to get reverted by owl bot. In this case here, we want to make changes to it (i.e. moving from `gcloud-devel` to JDST)

**Pull request merged**